packages/itwinui-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -102,6 +103,17 @@ export type ComboBoxProps<T> = {
    * Props to customize dropdown menu behavior.
    */
   dropdownMenuProps?: React.ComponentProps<'div'>;
+  /**
+   * End icon props.
+   */
+  endIconProps?: React.ComponentProps<typeof ComboBoxEndIcon>;
+  /**
+   * status message props.
+   */
+  statusMessageProps?: Pick<
```

Comment:
i've just realized that this may not actually be needed, since user can pass the actual `<StatusMessage>` component with all customizations

Reply:
Removed